File: Example/ObjcExample/ObjcExample/StyleChangeDemoController.m
//
//  StyleChangeDemoController.m
//  ObjcExample
//
//  Created by Animax Deng on 5/18/20.
//  Copyright © 2020 Animax. All rights reserved.
//

#import "StyleChangeDemoController.h"
#import <A_StyleInjection/A_StyleInjection.h>

@interface StyleChangeDemoController ()

@end

@implementation StyleChangeDemoController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)onClick:(id)sender {
    [[A_StyleManager shared] setupStyleSourceRepository:[[StylePlistProvider alloc] init:@"StyleSheet_2"]];
    [[A_StyleManager shared] applyStyle];
}

@end

File: Example/SwiftExample/SwiftExample/StyleChangeDemoController.swift
//
//  StyleChangeDemoController.swift
//  SwiftExample
//
//  Created by Animax Deng on 5/31/20.
//  Copyright © 2020 Animax. All rights reserved.
//

import UIKit
import A_StyleInjection

class StyleChangeDemoController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }

    @IBAction func onClick(_ sender: Any) {
        A_StyleManager.shared().setupStyleSourceRepository(StylePlistProvider("NewStyleSheet"))
    }
    
}
